monitoring/src/main/java/org/jlab/clas12/monitoring/tof_monitor.java

```diff
@@ -353,65 +381,85 @@ public tof_monitor(int reqrunNum, boolean reqTimeBased, boolean reqwrite_volatil
 		{"name":"timeResidual", "id":11,  "type":"float", "info":"time residual of the hit (in cm)"},
 	 */
-	public void fillDC(DataBank DCB, DataBank RunConfig){
+	public void fillDC(DataBank DCB, DataBank RunConfig, DataBank RecPart){
 		for(int r=0;r<DCB.rows();r++){
 			int s = DCB.getInt("sector",r)-1;
 			int sl = DCB.getInt("superlayer",r)-1;
 			float trkDoca = DCB.getFloat("trkDoca",r);
 			float timeResidual = DCB.getFloat("timeResidual",r);
 			float dDoca = DCB.getFloat("dDoca",r);
 			float time = DCB.getFloat("time",r);
+			int jitter = DCB.getInt("jitter",r);
 			double betacutvalue = 0.9;
 			double fitresidualcut = 1000; //microns
-			
+
 			//Determine alpha cut
 			double alphacutvalue = 30;
 		    double bFieldVal = (double) DCB.getFloat("B", r);
-		    int polarity = (int)Math.signum(RunConfig.getFloat("torus",0));	
+		    int polarity = (int)Math.signum(RunConfig.getFloat("torus",0));
 	        // alpha in the bank is corrected for B field.  To fill the alpha bin use the uncorrected value
 	        double theta0 = Math.toDegrees(Math.acos(1-0.02*bFieldVal));
 	        double alphaRadUncor = DCB.getFloat("Alpha", r)+ polarity*theta0;
 	        boolean alphacutpass = false;
 	        if(alphaRadUncor> -1*alphacutvalue &&  alphaRadUncor< alphacutvalue) {
 	            alphacutpass = true;
 	        }
-			
+		boolean leadingelectron = false;
+		int leadingparticlepid = RecPart.getInt("pid", 0);
+		if (leadingparticlepid == 11) leadingelectron = true;
+
 	        long timestamp = RunConfig.getLong("timestamp", 0);
-	        
+
 			// float field = DCB.getFloat("B",r); //removing per DC experts' request
-			if(s>-1&&s<6&&sl>-1&&sl<6){
+			if(s>-1&&s<6&&sl>-1&&sl<6 && leadingelectron){
 				// boolean otherregions = (sl<2 || sl>3);
 				// boolean region2 = ((sl==2||sl==3) && field<0.5);
 				// if (otherregions||region2) {
-		
+
 				//Fill Histograms with no extra cut
 				DC_residuals_trkDoca_nocut[s][sl].fill(trkDoca,timeResidual);
 				DC_residuals_nocut[s][sl].fill(timeResidual);
 				DC_time_nocut[s][sl].fill(time);
-				
+
 				//Add extra cuts on hits from DC4gui. TrkID, beta, alphacut, TFlight (maybe PID?, needs REC::Event here)
 				if (DCB.getByte("trkID", r) > 0 && DCB.getFloat("beta", r) > betacutvalue &&
 					 DCB.getFloat("TFlight",r) > 0 && alphacutpass == true
 						)
 				{
+
 					DC_residuals_trkDoca[s][sl].fill(trkDoca,timeResidual);
 					DC_residuals[s][sl].fill(timeResidual);
-					DC_time[s][sl].fill(time);	
-					
+					DC_time[s][sl].fill(time);
+					DC_jitterdist.fill(jitter);
+
 					if( timestamp%2 == 0) {//even time stamps
-						DC_time_even[s][sl].fill(time);	
+						DC_time_even[s][sl].fill(time);
+						//sector and superlayer need to go from 1-6
+						DC_hits_even_ts_sec_sl.fill(s+1,sl+1);
+						if (jitter == 0) {
+							DC_jitterzero_sec_sl.fill(s+1,sl+1);
+						}
+						if (jitter == 2 || jitter == -2 || jitter == 8 || jitter == -8) {
```

Comment:
Can you use `abs(jitter)`?

Reply:
Not at the moment. It might be that I want to do something different for negative jitter numbers. Need to check how it will be for a specific part of RGC data which will come soon for timelines and calibration.